pkg/acceptance/helpers/external_access_integration_client.go

```diff
@@ -0,0 +1,46 @@
+package helpers
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/stretchr/testify/require"
+)
+
+type ExternalAccessIntegrationClient struct {
+	context *TestClientContext
+	ids     *IdsGenerator
+}
+
+func NewExternalAccessIntegrationClient(context *TestClientContext, idsGenerator *IdsGenerator) *ExternalAccessIntegrationClient {
+	return &ExternalAccessIntegrationClient{
+		context: context,
+		ids:     idsGenerator,
+	}
+}
+
+func (c *ExternalAccessIntegrationClient) client() *sdk.Client {
+	return c.context.client
+}
+
+func (c *ExternalAccessIntegrationClient) CreateExternalAccessIntegration(t *testing.T, networkRuleId sdk.SchemaObjectIdentifier) (sdk.SchemaObjectIdentifier, func()) {
```

Comment:
nit: In cases like this, when we "mock" the implementation, please leave a TODO with a proper issue number to replace it in the future. In this case SNOW-1325215.

Reply:
Done.